recipes/tardis/meta.yaml

```
@@ -0,0 +1,107 @@
+{% set name = "tardis-sn" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
```

These should all be [`host` dependencies instead of `build`dependencies](https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html?highlight=selectors#requirements-section):
```suggestion
  host:
```